### PR TITLE
Update to ansible-lint 5.0 and add yamllint 

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Also includes fixes for container builds

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM centos:7
 
 RUN echo "tsflags=nodocs" >> /etc/yum.conf && \
     yum -y install epel-release git && \
-    yum -y install python2-pip && \
+    yum -y install python-pip && \
```
Comment (evgeni) [inline Dockerfile:5]: this *shouldn't* be needed (but also doesn't hurt)

File: obal/data/playbooks/bump-release/bump-release.yaml [hunk 2]
```diff
@@ -12,3 +12,4 @@
       command: "rpmdev-bumpspec {{ spec_file_path | basename }} -c \"{{ changelog }}\""
       args:
         chdir: "{{ spec_file_path | dirname }}"
+      changed_when: false
```
Comment (evgeni) [inline obal/data/playbooks/bump-release/bump-release.yaml:15]: I'd argue this is always changing, as bumpspec will allways bump?

**Comment by evgeni** [comment]
I didn't mean you shall not install `python-pip` at all, just that it doesn't matter if you call it `python-pip` or `python2-pip` ;)

**Comment by ehelms** [comment]
> I didn't mean you shall not install `python-pip` at all, just that it doesn't matter if you call it `python-pip` or `python2-pip` ;)

I see that now!

**Review by evgeni — APPROVED** [review]

**Pull request merged**